common/src/main/java/com/microsoft/hydralab/performance/parsers/WindowsBatteryResultParser.java

```diff
@@ -43,11 +43,22 @@ public class WindowsBatteryResultParser implements PerformanceResultParser {
 
     @Override
     public PerformanceTestResult parse(PerformanceTestResult performanceTestResult) {
+        try {
+            // Wait 20 seconds for the results to be completely written to the hard disk.
+            Thread.sleep(20 * 1000);
```

Comment:
I am wondering if it will affect the running time of test tasks. As the `parse` method is invoked synchronously.